### ISSUE TITLE
rounding of values displayed in wallet summary/panel

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#4314]
## Description 
I had an empty wallet (0 BAT balance). I got an Ad grant for .2 BAT. Once I accepted, I saw my wallet balance displaying as .3 BAT. Upon further inspection, it seems my Ad grant was actually for .25 BAT, but the UI only shows one decimal place (on both the 'it's your lucky day' screen and in the summary section). 

The problem isn't necessarily the only showing one decimal place, it's that the displayed/rounded value is inconsistent. So in the summary/panel my Ad grant amount is rounded down from .25 to .2, but in my wallet balance area it's rounded up to .3. This is confusing.

## Steps to Reproduce

   1. Using a wallet with 0 BAT balance, receive an Ad grant that is something like .25 BAT.
   2. Perform captcha and view the resulting 'It's your lucky day' screen. It says .2 BAT.
   3. Close this screen and view the wallet panel/summary.

## Actual result:
Wallet balance is .3 BAT but lower summary section says .2 BAT.
<img width="396" alt="Screen Shot 2019-05-06 at 8 35 59 AM" src="https://user-images.githubusercontent.com/28145373/57226057-0e01d480-6fdc-11e9-8647-96cf2c3b29c5.png">

## Expected result:
If we're only going to display one decimal, it would be nice if both wallet balance and sections in the summary used the same rounding.

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 0.63.48 Chromium: 74.0.3729.108 (Official Build) (64-bit)
-- | --
Revision | daaff52abef89988bf2a26091062160b1482b108-refs/branch-heads/3729@{#901}
OS | Mac OS X

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? unsure
- Can you reproduce this issue with the dev channel? unsure
- Can you reproduce this issue with the nightly channel? unsure

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? n/a
- Does the issue resolve itself when disabling Brave Rewards? n/a
- Is the issue reproducible on the latest version of Chrome? n/a

## Miscellaneous Information:
I visited https://ledger.mercury.basicattentiontoken.org/v2/wallet/xyz where xyz is the wallet paymentId to confirm my wallet balance.

**Comment by GeetaSarvadnya** [comment]
Reproduced on windows `0.64.68`

![image](https://user-images.githubusercontent.com/38657976/57236691-48528c80-7043-11e9-9d27-b5dea9218001.png)

**Comment by kjozwiak** [comment]
Here's another example of the above issue that I ran into today. Brave is rounding by total from `0.97 BAT` to `1 BAT`. Because all my main rewards panels are displaying 1 BAT, I figured I still had enough to send one more tip. However, I ran into the `insufficient funds` error. Noticed that the tipping banner is displaying `0.97 BAT`. This will most likely confuse some users. Example:

<img width="705" alt="Screen Shot 2019-06-06 at 12 17 21 PM" src="https://user-images.githubusercontent.com/2602313/59050482-6ab11300-8858-11e9-92f3-b2d83b6ffeb9.png">
